### PR TITLE
Add snyk support to ods-core

### PR DESCRIPTION
**Pull request by stitakis**
adds small changes

### CODE REVIEW

**Comment by stitakis** [comment]
#171

**Comment by stitakis** [comment]
@rattermeyer thanks for spotting it... for now I have added the missing documentation to the adoc file. Regarding the question... I would vote for creating a new issue for avoiding docs duplication between adocs and README.

File: jenkins/slave-base/README.md [hunk 1]
```diff
@@ -11,3 +11,4 @@ The base jenkins slave used by all ODS [builder slaves](https://github.com/opend
 1. [OWASP scan plugin](https://dl.bintray.com/jeremy-long/owasp/)
 1. Creates proxy awareness when `HTTP_PROXY` is injected during build pahse
 1. [Tailor](https://github.com/opendevstack/tailor) - on top of `openshift CLI` to provide infrastrucure as code
+1. Installs [Snyk Security Scan CLI](https://github.com/snyk/snyk) when 'SNYK_DISTRIBUTION_URL' is injected during build phase
```
Comment (michaelsauter) [inline jenkins/slave-base/README.md]: Use backticks to make it render in code blocks. See difference: 'SNYK_DISTRIBUTION_URL' vs `SNYK_DISTRIBUTION_URL`.

Comment (stitakis) [reply jenkins/slave-base/README.md]: @michaelsauter done

**Pull request merged**